static/app/components/events/interfaces/sourceMapsDebuggerModal.tsx

```diff
@@ -1882,3 +1934,17 @@ const ScrapingSymbolificationErrorMessage = styled('p')`
   padding-left: ${space(1)};
   margin-top: -${space(1)};
 `;
+
+const DebuggerSectionContainer = styled(Flex)`
```

Comment:
just put all the styles here instead of mixing styles on top of Flex

Reply:
should Flex be used entirely without restyling? Do we have any guidelines on this?

Reply:
avoid styling styled components in general

Reply:
I understand...just noticed that we have this all over the app. Was this discussed in the TSC? if not, we should :)